onnxscript/rewriter/basic_rules.py

```diff
@@ -311,6 +384,7 @@ def basic_optimization_rules() -> RewriteRuleSet:
             cast_cast_rule,
             cast_identity_rule,
             expand_identity_rule,
+            flatten_to_reshape_rule,
```

Comment:
Better specify this rule is order sensitive to reshape_reshape_rule

Reply:
@titaiwangms How? with a comment?

Reply:
A comment would be good for now I think

Reply:
Done